dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -124,6 +122,12 @@ with a semicolon (';') and may contain dots.
 
 Example: "left.png;right.jpg")code",
             DALI_STRING_VEC)
+    .AddOptionalArg("index_paths", // to be changed
```

Comment:
1. It will fail in the linter (too few spaces before comment).
2. What does this comment even mean? Do we want to change the argument name? If so, then let's better do it ASAP, before anyone attempts to use it.